app/models/group.rb

```diff
@@ -1,4 +1,6 @@
 class Group < ApplicationRecord
+  belongs_to :organisation
```

Comment:
Is it worth adding the reverse `has_many :groups` to `Organisation` now or wait until we need it?

Reply:
I can't immediately think of a use for the reverse relationship, so I vote leave it out?